src/components/back-to-top/_back-to-top.scss

```diff
@@ -0,0 +1,37 @@
+.ons-back-to-top {
+  &__description {
+    margin-left: 0.2rem;
+  }
+
+  &__enabled {
+    background: none;
+    width: fit-content;
+    padding: 0.5em 0;
+
+    > a {
```

Comment:
and you can use .ons-back-to-top__link here, but I don't think this bit is having any affect